Sources/xcproj/PBXContainerItemProxy.swift

```diff
@@ -61,9 +61,9 @@ public class PBXContainerItemProxy: PBXObject, Hashable {
     public required init(from decoder: Decoder) throws {
         let container = try decoder.container(keyedBy: CodingKeys.self)
         self.containerPortal = try container.decode(.containerPortal)
-        let proxyTypeString: String = try container.decode(.proxyType)
-        self.proxyType = UInt(proxyTypeString).flatMap({ProxyType(rawValue: $0)}) ?? .other
-        self.remoteGlobalIDString = try container.decode(.remoteGlobalIDString)
+        let proxyTypeString: String? = try container.decodeIfPresent(.proxyType)
```

Comment:
I noticed when the decoding changes happened there are a whole bunch of these cases where something is decoded into a string and then flatMaped into a UInt. Is there a specific reason for that, as opposed to decoding directly to a UInt?

Reply:
Does this not work?
```
self.proxyType = try container.decodeIfPresent(UInt.self, forKey: . proxyType).flatMap(ProxyType.init) ?? .other
```

Reply:
Yeah the decoder throws because doesn't know how to convert those values into integers @yonaskolb

Reply:
Ah ok, makes sense